OSDOCS-17013-bm-upi-4: CQAs bare metal UPI doc (#104267)
openshift/openshift-docs: modules/installation-minimum-resource-requirements.adoc

[id="installation-minimum-resource-requirements_{context}"]
= Minimum resource requirements for cluster installation

[role="_abstract"]

> When creating a cluster, the cluster must meet minimum requirements so that the cluster runs as expected.

Reviewer: ```suggestion
Each created cluster must meet minimum requirements so that the cluster runs as expected.
```
perhaps simpler?
